ROADMAP.md

```diff
@@ -1,16 +1,37 @@
-# KServe 2023 Roadmap
+# KServe 2024-2025 Roadmap
+## Objective: "Support GenAI inference"
+- LLM Serving Runtimes
+   * Support Speculative Decoding with vLLM runtime.
+   * Support LoRA adapters.
+   * Support LLM Serving runtimes for TensorRT-LLM, TGI and provide benchmarking comparisions.
+   * Support multi-host, multi-GPU inference runtime.
+
+- LLM Autoscaling
+   * Support Model Caching with automatic PV/PVC provisioning.
+   * Support Autoscaling settings for serving runtimes.
+   * Support Autoscaling based on custom metrics.
+
+- LLM RAG Pipeline Orchestration
```

Comment:
Do we care about Agtent workflows and tool based ones? cc @pierdipi

Reply:
I'd be happy to work with @pierDipi and others interested on supporting agentic workflows here (if that makes sense)

Reply:
I believe it's better to discuss at an high level what that means in an issue and whether it's in scope for kserve or not before adding it to the roadmap doc?

Reply:
Opened an issue here https://github.com/kserve/kserve/issues/3829

Reply:
added to the roadmap, I think this should be in the scope of KServe to orchestrate RAG/Agent workflow